jobfair-frontend/src/pages/ProfilePage/Company/index.jsx

```diff
@@ -0,0 +1,289 @@
+import React from 'react';
+import {Controller, useFieldArray, useForm} from "react-hook-form";
+import {COMPANY_DEFAULT_MODEL} from "./CompanyDefaultModel";
+import {benefitConst, SizeConst, SubCategories} from "./CompanyProfileConstant";
+import {Box, Paper} from "@mui/material";
+import Select from "react-select";
+import styles from "./index.styles.scss";
+import {updateCompanyProfile} from "../../../services/userService";
+import {ErrorMessage} from "@hookform/error-message";
+
+const data = {
```

Comment:
@CaoSon1909  please move this data and create file MockData.js to use data for mocking